services/dateHelper.js

```diff
@@ -117,6 +117,14 @@ function validInputs(month, day, year) {
 }
 
 function checkDateValid(month, day, year) {
+  const invalidDateInput = [month, day, year].some((element) => {
+    if (element !== "") {
+      return !/\d+$/.test(element)
+    } else return false
+  })
```

Comment:
I'm sorry, my lack of JavaScript knowledge is inhibiting my ability to understand this.

Reply:
It is using a Regex expression to loop through the array and test for when an input has something other then numbers and stops looping when there is an input other than numbers but not the default empty input